### PR TITLE
490 bound the size of the taskqueues lh server

### PR DESCRIPTION
**Pull request by eduwercamacaro**

### CODE REVIEW

File: server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java [hunk 1]
```diff
@@ -84,6 +85,23 @@ protected <U extends Message, T extends CoreGetable<U>> List<GetableToStore<U, T
...
+        return store.range(start, end, Tag.class);
+    }
+
+    /**
+     * Only use this method if you need to retrieve a Storeable
+     * @param storeKey Storeable's storekey
+     * @param cls Storeable's class
+     * @return Null if not found
+     */
+    public <U extends Message, T extends Storeable<U>> T get(String storeKey, Class<T> cls) {
```
Comment (eduwercamacaro) [inline server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java:101]: @coltmcnealy-lh  This is leaking the implementation detail. Any suggestion here? see `OneTaskQueue#rehydrateFromStore`

Comment (coltmcnealy-lh) [reply server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java:101]: Can we just rename it to `getStoreable()`? I don't like overloading the `get()` method, which is used to get a `Getable`.

Comment (coltmcnealy-lh) [reply server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java:101]: Actually, try this:

```
public ScheduledTaskModel getScheduledTask(TaskRunIdModel scheduledTaskId);
```

That still leaks, but it leaks a lot less.

**Review by coltmcnealy-lh — APPROVED** [review]

File: server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java (hunk 1 above)
Comment (eduwercamacaro) [reply server/src/main/java/io/littlehorse/server/streams/storeinternals/ReadOnlyGetableManager.java:101]: That leaks the business application into the store abstraction. ;)

**Pull request merged**